repo/packages/A/avi/3/package.json

```diff
@@ -0,0 +1,28 @@
+{
+  "packagingVersion": "2.0",
+  "name": "avi",
+  "description": "Avi Vantage ADC. An example for this package's usage can be found at https://github.com/dcos/examples/tree/master/avi",
+  "version": "3.0",
+  "framework": false,
+  "scm": "https://github.com/mesosphere/avi-lb",
+  "maintainer": "siva@avinetworks.com",
+  "tags": ["loadbalancer", "service-discovery", "reverse-proxy", "adc", "analytics", "microservices", "gslb"],
+  "preInstallNotes": "We recommend a minimum of 4 CPUs/12 GB of RAM available for the Avi Controller Service; and 1 CPU/2 GB RAM for Avi ServiceEngine.",
```

Comment:
@ryadav88 I have added the preview note. I am able to launch the service fine with defaults.